rclpy/package.xml

```diff
@@ -48,9 +50,18 @@
   <test_depend>test_msgs</test_depend>
 
   <doc_depend>python3-sphinx</doc_depend>
+  <doc_depend>python3-sphinx-autodoc-typehints</doc_depend>
```

Comment:
After discussion, this doesn't seem to be necessary, so let's just remove it.

Reply:
Done [609605c](https://github.com/ros2/rclpy/pull/1103/commits/609605cf7340335beb0c256e1f3b77d9524d94cd)